### DISCUSSION TITLE
mv为什么无法播放

### DISCUSSION THREAD

**Discussion by Harlouavicii** [qier222/YesPlayMusic#1375] [Q&A]
![image](https://user-images.githubusercontent.com/94124500/156737789-597ebc38-8514-4b19-97dc-501a6a52d12a.png)

**Comment by memorydream** [comment]
虽然不知道你在播放的mv是什么，但是mv播放功能在我这里正常工作
![image](https://user-images.githubusercontent.com/34763046/156901917-9fa7ad6b-3b9b-4edf-aafc-6c5ecd7ef79b.png)

**Comment by Harlouavicii** [comment]
需要在设置中开启或关闭什么功能吗？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "qier222/YesPlayMusic"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月6日(星期天) 上午6:32
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [qier222/YesPlayMusic] mv为什么无法播放 (Discussion #1375)





 
虽然不知道你在播放的mv是什么，但是mv播放功能在我这里正常工作
 
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by memorydream** [reply]
不需要任何设置
